src/routes/docs/products/ai/integrations/elevenlabs/+page.markdoc

```diff
@@ -0,0 +1,358 @@
+---
+layout: article
+title: Integrating ElevenLabs into Appwrite
+description: Learn how to integrate ElevenLabs into your Appwrite project.
+difficulty: beginner
+readtime: 15
+back: /docs/products/ai/audio-processing
+---
+
+Integrating ElevenLabs into your Appwrite project is simple. This tutorial will guide you through the process of setting up the ElevenLabs API and integrating it into your Appwrite project.
+
+## Prerequisites
+- An Appwrite Project
+- An ElevenLabs API Key
+
+{% section #step-1 step=1 title="Create new function" %}
+Head to the [Appwrite Console](https://cloud.appwrite.io/console) then click on **Functions** in the left sidebar and then click on the **Create Function** button.
+
+{% only_dark %}
+![Create function screen](/images/docs/functions/dark/template.png)
+{% /only_dark %}
+{% only_light %}
+![Create function screen](/images/docs/functions/template.png)
+{% /only_light %}
+
+1. In the Appwrite Console's sidebar, click **Functions**.
+2. Click **Create function**.
+3. Under **Connect Git repository**, select your provider.
+4. After connecting to GitHub, under **Quick start**, select the **Node.js** starter template.
+5. In the **Variables** step, tick the box to **Generate API key on completion**.
+6. Follow the step-by-step wizard and create the function.
+{% /section %}
+
+{% section #step-2 step=2 title="Add Fetch" %}
+Once your function is created, navigate to the freshly created repository and either clone it to your local machine or launch a GitHub Codespace.
```

Comment:
> or launch a GitHub Codespace. 

Is this how what we suggest in other function guides as well? Else lets remove it

Reply:
Removed